jre/guide/java/java_jre_lock_untrusted_sources/oval/jre.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+  <definition class="compliance" id="java_jre_lock_untrusted_sources" version="1">
+    <metadata>
+      <title>Lock Execution of Signed Java Applets From Untrusted Sources Setting</title>
+      <affected family="unix">
+        <product>Java Runtime Environment</product>
+      </affected>
+      <description>The dialog to enable users to grant permissions to execute 
+      signed content from an un-trusted authority must be locked.</description>
```

Comment:
@carlosmmatos can you expand the description like what is done in the others to tell people how this is done?